subt/push.py

```diff
@@ -12,6 +12,7 @@ class Push:
 
     def __init__(self, config, bus):
         bus.register()
+        self.bind = config.get("bind", False)
```

Comment:
at first `self.bind` correlated to some `bind()` function ... but let's leave it